twins/service.go

```diff
@@ -154,6 +156,10 @@ func (ts *twinsService) AddTwin(ctx context.Context, token string, twin Twin, de
 	id = twin.ID
 	b, err = json.Marshal(twin)
 
+	if err := ts.twinCache.Save(ctx, twin); err != nil {
```

Comment:
I think you can directly `return ts.twinCache.Save(ctx, twin)` here

Reply:
I wouldn't be returning an empty Twin then, which is probably not important.

Reply:
You are right, this does not return 2 values, I did not see.

Sorry for this remark, please ignore it.